mk/make_rc_defines.sh

```diff
@@ -2,8 +2,11 @@
 
 ML_USER=`id | awk -F')' '{ print $1 }' | awk -F'(' '{ print $2 }'`
 ML_VERSION_STR=`cat $CPP_SRC_HOME/gradle.properties | grep '^elasticsearchVersion' | awk -F= '{ print $2 }' | xargs echo`
+if [ -n "$VERSION_QUALIFIER" ] ; then
+    ML_VERSION_STR="$ML_VERSION_STR-$VERSION_QUALIFIER"
+fi
 if [ "$SNAPSHOT" != no ] ; then
-    ML_VERSION_STR="$ML_VERSION_STR"-SNAPSHOT
+    ML_VERSION_STR="$ML_VERSION_STR-SNAPSHOT"
```

Comment:
Was this intended? I don't think you want to move the closing quote

Reply:
I moved it to be more like the new line for the qualifier a little higher up.  It doesn't make any difference functionally.  Happy to put it back to how it was though if you think it looked better before.